src/python/pants/backend/docker/rules_test.py

```diff
@@ -0,0 +1,213 @@
+# Copyright 2021 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
And another naming nit: Split this up into `docker_binary_test.py`, `build_context_test.py`, so that the unit test files mirror the files of the code they test?